vision/emergent_object/emergent_object.py

```diff
@@ -0,0 +1,61 @@
+import torch
+
+# You may have to install:
+#   pandas
+#   torchvision
+#   tqdm
+#   seaborn
+
+MODEL_PATH = "vision/emergent_object/best.pt"
```

Comment:
Since this constant is only used in `emergent_object_model()`, constant `MODEL_PATH` should be moved to that function.